packages/wallet/backend/src/gatehub/service.ts

```diff
@@ -18,8 +18,8 @@ export class GateHubService {
     private gateHubClient: GateHubClient,
     private logger: Logger,
     private env: Env,
-    private accountService?: AccountService,
-    private walletAddressService?: WalletAddressService
+    private accountService: AccountService,
```

Comment:
`gatehubService.setupSandboxCustomer`
still checks existance of accountService and walletAddressService, which are no longer optional